deployment/docker-build/docker-compose.yml

```
@@ -8,22 +8,39 @@ volumes:
   pyaleph-mongodb:
 
 services:
-  p2pd:
+
+  rabbitmq:
     restart: always
-    image: alephim/jsp2pd:0.10.2-1.0.0
+    image: rabbitmq:3.10.7-management
     networks:
       - pyaleph
     environment:
-      PRIVATE_KEY_FILE: "/etc/jsp2pd/keys/serialized-node-secret.key"
-      LISTEN_MADDR: "/ip4/0.0.0.0/tcp/4030"
-      HOST_MADDRS: "/ip4/0.0.0.0/tcp/4025"
-      PUBSUB: "true"
-      PUBSUB_ROUTER: "floodsub"
+      RABBITMQ_DEFAULT_USER: guest
+      RABBITMQ_DEFAULT_PASS: guest
+    ports:
+      - "5672:5672"
+      - "15672:15672"
```

Do not expose management interface on a public interface.
```suggestion
      - "127.0.0.1:15672:15672"
```

---

Good catch, I'll clean all the others in the file as well.